.cargo/config.toml

```diff
@@ -5,6 +5,12 @@ build-esp32c3 = "build --release --target riscv32imc-unknown-none-elf --features
 build-esp32c6 = "build --release --target riscv32imac-unknown-none-elf --features esp32c6"
 build-esp32s2 = "build --release --target xtensa-esp32s2-none-elf --features esp32s2"
 build-esp32s3 = "build --release --target xtensa-esp32s3-none-elf --features esp32s3"
+run-esp32   = "run --release --target xtensa-esp32-none-elf --features esp32"
```

Comment:
This would allow doing `cargo run-esp32`, `cargo run-esp32c6` and it would build/flash the proper target